LPS-184523 Use try() when prepareStatement connection is used

gustavolimav opened the pull request:
Resend: https://github.com/brianchandotcom/liferay-portal/pull/137129#issuecomment-1614249982

Commit:
LPS-184523 LPS-186827 search-experiences-web: Blueprint Options widget is storing ERC of the Blueprint instead of its ID

https://issues.liferay.com/browse/LPS-186827

Commit:
LPS-184523 LPS-186826 search-experiences-service: Add _contributeSXPBlueprintExternalReferenceCode method

that method allow us to use search.experiences.blueprint.erc in searchContext to apply a blueprint to the search

https://issues.liferay.com/browse/LPS-186826

Commit:
LPS-184523 LPS-186826 search-experiences-service: Refactor code

Commit:
LPS-184523 LPS-187222 search-experiences-service: Add ERC attribute to
suggestions

https://liferay.atlassian.net/browse/LPS-187222

Commit:
LPS-184523 LPS-187223 portal-search-web: Use suggestions with ERC instead of ID

https://liferay.atlassian.net/browse/LPS-187223

Commit:
LPS-184523 LPS-188093 search-experiences-service: Use Id insted of ERC in Blueprints Collection Provider

https://liferay.atlassian.net/browse/LPS-188093

Commit:
LPS-184523 LPS-186828 search-experiences-service: Add upgrade process to blueprint options widget

https://issues.liferay.com/browse/LPS-186828

Commit:
LPS-184523 LPS-188356 search-experiences-service: Add upgrade Process to suggestionsContributorConfigurations

https://liferay.atlassian.net/browse/LPS-188356

brianchandotcom committed:
LPS-184523 Spell it out

brianchandotcom committed:
LPS-184523 SF

brianchandotcom committed:
LPS-184523 Can we sort this?

brianchandotcom committed:
LPS-184523 Buy space

brianchandotcom committed:
LPS-184523 Simplify

brianchandotcom committed:
LPS-184523 Inline

brianchandotcom committed:
LPS-184523 SF

Commit:
LPS-184523 Use try() when prepareStatement connection is used

try (PreparedStatement preparedStatement1 = connection.prepareStatement(

Is not the same as

PreparedStatement preparedStatement1 = connection.prepareStatement(

The former will auto close. The latter will not clean itself up.

liferay-continuous-integration commented:
Closing pull request because all `liferay-portal` pullrequests sent to Brian Chan must be sent by using `ci:forward` on a pull request that was sent to someone else.

gustavolimav commented:
ci:reopen

Commit:
LPS-184523  LPS-189331 search-experiences-service: Implement priority to apply blueprint attributes

its similar to chain responsability pattern but simple

https://liferay.atlassian.net/browse/LPS-189331

liferay-continuous-integration commented:
Closing and locking pull request because pull requests sent to this user may not be updated. Please resend this pull request.
